### ISSUE TITLE
NVDA screen reader reads out in wrong order when you tab away from the magnet before dropping it

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/faradays-law#210]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/QA/issues/581. Found with NVDA on Firefox.
When you have grabbed the magnet, then tab away to the Voltmeter checkbox, without pressing space to first drop the magnet, it seems to read things out of order. The first thing it reads out is a description for the magnet you just left. It then reads out the Voltmeter description. Finally, it says that the magnet was released.
**Visuals**
https://drive.google.com/file/d/14KqLfdJH8u31J0_cxoFhCIcQJwOtgxFl/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Faraday's Law‬
URL: https://phet-dev.colorado.edu/html/faradays-law/1.4.0-rc.1/phet/faradays-law_all_phet.html
Version: 1.4.0-rc.1 2020-12-02 22:52:22 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:83.0) Gecko/20100101 Firefox/83.0
Language: en-US
Window: 1280x687
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
I am guessing the order of operations is
 - Press tab on grabbed magnet
 - blur event fires on the GrabDragInteraction, changing its accessible content. NVDA then reads the new label.
 - "Released" alert is added to back of utterance queue.
 - focus lands on the checkbox, nvda reads the label
 - "Released" alert is spoken after utteranceQueue delay.

Expect this to happen elsewhere, going to check on a few sims.

**Comment by jessegreenberg** [comment]
@jbphet and I looked at this issue. It will probably be difficult to solve with our limitation of ordering aria-live related alerts and what gets read out by the screen reader on focus. IT is also not specific to faradays-law. We are going to move this to a more common repo to track, and see if anything can be done. But it should not block the upcoming faradays-law release.

**Comment by jessegreenberg** [comment]
Moving to scenery-phet.

**Comment by jessegreenberg** [comment]
Issue moved to [phetsims/scenery-phet #649](https://github.com/phetsims/scenery-phet/issues/649) via [**ZenHub**](https://www.zenhub.com/)
